recipes/wasmtime/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest)
+
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(wasmtime REQUIRED)
+
+add_executable(example example.cpp)
+target_link_libraries(example PRIVATE wasmtime::wasmtime)
+target_compile_options(example PRIVATE ${CONAN_COMPILE_DEFINITIONS_WASMTIME})
```

Comment:
we be included with the targets

```suggestion
```

Reply:
I cannot do this, because `wasmtime::wasmtime` do not have compile definitions for this target ...
By some reason `conan` do not add `target_compile_definitions(...)` for this target

Reply:
Also very strange behavior:
```cmake
set(CONAN_DEFINES_WASMTIME "-D/DWASM_API_EXTERN="
			"-D/DWASI_API_EXTERN=")
set(CONAN_BUILD_MODULES_PATHS_WASMTIME )
# COMPILE_DEFINITIONS are equal to CONAN_DEFINES without -D, for targets
set(CONAN_COMPILE_DEFINITIONS_WASMTIME "/DWASM_API_EXTERN="
			"/DWASI_API_EXTERN=")
```
`CONAN_COMPILE_DEFINITIONS_WASMTIME` generated properly, but `CONAN_DEFINES_WASMTIME` is incorrect

Reply:
With https://github.com/conan-io/conan-center-index/pull/6083/files#r670901897, this should be correct now.

Reply:
No it is not, see comment above for it